changelog.md

```diff
@@ -1,23 +1,25 @@
 # Zepben Python SDK
 ## [0.38.0] - UNRELEASED
 ### Breaking Changes
-* None.
+* `connect_with_secret()` and `connect_with_password()` will no longer create a `ZepbenTokenFetcher` directly from kwargs.
 
 ### New Features
-* None.
+* Added support for `getMetadata()` gRPC calls on `CustomerConsumerClient`, `DiagramConsumerClient`, and `NetworkConsumerClient`.
 
 ### Enhancements
-* None.
+* `GrpcChannelBuilder` tests the connectivity of newly created channels before returning them to the user. This is done by calling `getMetadata()` against all
+known services, the channel is returned after the first successful response. Any connectivity errors will be propagated to the user. If no connectivity errors
+are encountered but no successful response is received from the known services, a `ConnectionException` is thrown.
 
 ### Fixes
 * None.
 
 ### Notes
 * None.
 
-## [0.37.0] - UNRELEASED
+## [0.37.0] - 2023-11-14
```

Comment:
@kgreav Looks like the release action isn't versioning each set of notes in the changelog. Something to be investigated, and you can assign the task to me.

Reply:
Yep fairly sure it's already been fixed by @alourie.